src/main/resources/db/changelog/ddl/0008_EIP1-2833_alter_delivery.xml

```diff
@@ -0,0 +1,48 @@
+<?xml version="1.0" encoding="UTF-8"?>
+<databaseChangeLog
+    xmlns="http://www.liquibase.org/xml/ns/dbchangelog"
+    xmlns:xsi="http://www.w3.org/2001/XMLSchema-instance"
+    xsi:schemaLocation="http://www.liquibase.org/xml/ns/dbchangelog
+      http://www.liquibase.org/xml/ns/dbchangelog/dbchangelog-4.1.xsd">
+
+    <changeSet author="nathan.russell@valtech.com" id="0008_EIP1-2833_alter_delivery - add address_format column" context="ddl">
+
+        <addColumn tableName="delivery">
+            <column name="address_format" type="varchar(20)" value="UK">
+                <constraints nullable="false" />
+            </column>
+        </addColumn>
+
+        <rollback>
+            <dropColumn
+                tableName="delivery"
+                columnName="address_format"
+            />
+        </rollback>
+    </changeSet>
+
+    <changeSet author="nathan.russell@valtech.com" id="0008_EIP1-2833_alter_delivery - rename delivery_method column" context="ddl">
```

Comment:
Rename delivery_method to address_type; and set all values to REGISTERED as thats the only value we support at the moment (previous column value was DELIVERY)